rp-events/template.yaml

```diff
@@ -14,13 +14,9 @@ Parameters:
       - production
 
 Conditions:
-  IsProduction: !Equals [!Ref Environment, 'production']
-  NotTestEnvironment:
-    !Or [
-      !Equals [!Ref Environment, production],
-      !Equals [!Ref Environment, integration]
-    ]
-  TestEnvironment: !Not [Condition: NotTestEnvironment]
+  IsIntegratedEnvironment: !And
```

Comment:
Sorry to be pedantic (why break the habit of a lifetime though), but it looks like you've added this condition (and replaced what was there) but haven't used it at all?

Reply:
apologies, this was meant to go before I opened the PR. I've removed all the conditions as none of them were being used.